msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -41,7 +41,7 @@ To add tabs to an existing meeting:
 1. In your calendar, select a meeting to which you want to add a tab.
 1. Select the **Details** tab and select <img src="~/assets/images/apps-in-meetings/plusbutton.png" alt="Plus button" width="30"/>. The tab gallery appears.
 
-    <img src="../assets/images/apps-in-meetings/PreMeeting.png" alt="Pre-meeting experience" width="900"/>
+    :::image type="content" source="~/assets/images/apps-in-meetings/Pre-Meeting-002.png" alt-text="Pre-meeting app experience" :::
```

Comment:
I don't know if the space at the end will impact anything but let's be cognizant of the syntax. See https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=main#standard-conceptual-images-using-docs-markdown-recommended

There are no spaces around `:::`.

CC @surbhigupta12 .